interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
+<?php $collectThis=collectValidationPageRules("add_edit_issue",$_SERVER['PHP_SELF'])?>
```

Comment:
I see what you mean but since option_id is a part of a key in the db there cannot be multiple records with the same ID the id must be created per form.By collecting the path we can get all the validation issued on this page. since I am already returning the option_id in page_name this way the developers can use it to validate whatever form they need with this id. 

So all I need to do change the field page_name context  value to include the page_name+form_id like "add_edit_issue"+"#"+"form_name"
this way I can use explode function on the "#" and collect the form_name from it and use it .
what do you think will it be more functional like this?